homeassistant/components/aurora_abb_powerone/coordinator.py

```diff
@@ -42,34 +46,22 @@ def __init__(
             update_interval=SCAN_INTERVAL,
         )
 
-    def _update_data(self) -> dict[str, float]:
+    def _update_data(self) -> dict[str, Any]:
         """Fetch new state data for the sensors.
 
         This is the only function that should fetch new data for Home Assistant.
         """
-        data: dict[str, float] = {}
         self.available_prev = self.available
         retries: int = 3
+        data = {}
```

Comment:
Why not typehint here to aid maintenance
```suggestion
        data: dict[str, Any] = {}
```